Digipost.Api.Client.DataTypes/Proof/Proof.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml;
+
+namespace Digipost.Api.Client.DataTypes.Proof
+{
+    public class Proof : IDataType
+    {
+        public Proof(string utstederVisningsnavn, GyldighetsPeriode gyldighetsPeriode, Bruker bevisBruker, string tittel)
+        {
+            UtstederVisningsnavn = utstederVisningsnavn;
+            GyldighetsPeriode = gyldighetsPeriode;
+            BevisBruker = bevisBruker;
+            Tittel = tittel;
+        }
+
+        /// <summary>
+        ///     Name of the authorizer
+        /// </summary>
+        public string UtstederVisningsnavn { get; set; }
```

Comment:
Synes alle felter i denne burde ha engelske navn siden vi kaller det for `Proof`. Dette krever vel en endring i data-types først, men synes det burde bli gjort før denne merges.
Ref https://github.com/digipost/digipost-data-types/pull/49/files#r340070597

Reply:
Kan sette igang med å oversette her

Reply:
Fikset i 8c23d67